.github/workflows/deployment-staging.yml

```diff
@@ -23,8 +23,8 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Deploy on Staging Website
-        env:
-          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        env: 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           npm install
-          npm run deploy:staging -- --repo=https://${{secrets.GH_TOKEN}}@github.com/CATcher-org/CATcher-staging.git
\ No newline at end of file
+          npm run deploy:staging -- --repo=https://x-access-token:$GITHUB_TOKEN@github.com/seanlowjk/CATcher-staging.git
```

Comment:
```suggestion
          npm run deploy:staging -- --repo=https://x-access-token:$GITHUB_TOKEN@github.com/CATcher-org/CATcher-staging.git
```
Think you forgot to revert this, after testing !

Reply:
ah oops thank you ><